twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -450,6 +432,7 @@ scrooge_scala_import = rule(
     attrs = {
         "thrift_jars": attr.label_list(allow_files = [".jar"]),
         "scala_jars": attr.label_list(allow_files = [".jar"]),
+        "deps": attr.label_list(),
```

Comment:
Is this a new attribute? If so do we want a test to show how to use it? Also will help us not delete it by mistake

Reply:
Good point. I'll see what I can wire up.

Reply:
Err wait a minute. I actually don't think this attribute is necessary. I think I got confused between `scrooge_scala_import` and `scrooge_scala_library`. I'll get back to you.